src/mapkit.cs

```diff
@@ -327,6 +331,17 @@ interface MKMapItem : NSSecureCoding
 		[Export ("openMapsWithItems:launchOptions:"), Internal]
 		bool _OpenMaps ([NullAllowed] MKMapItem [] mapItems, [NullAllowed] NSDictionary launchOptions);
 
+		[iOS (13, 2), NoMac, NoTV, NoWatch]
+		[Introduced (PlatformName.UIKitForMac, 13, 2)]
```

Comment:
Wasn't sure about Catalyst - according to Apple docs, this will be supported in Catalyst: https://developer.apple.com/documentation/mapkit/mkmapitem/3395859-openinmapswithlaunchoptions?language=objc

However, it was not explicitly listed as supported in the diffs.